syntax = "proto3";

package msg;


// The boat service definition.
service BoatService {
    // Sends a greeting
    rpc SayHello (BoatRequest) returns (BoatResponse) {}

    rpc SayHelloStreamReply (BoatRequest) returns (stream BoatResponse) {}
}
  
// The request message containing the user's name.
message BoatRequest {
    string name = 1;
}

// The response message containing the greetings
message BoatResponse {
    string message = 1;
}
  